accounts/tasks.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from celery import shared_task
+from django.apps import apps
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.utils.translation import ugettext as _
+
+from .models import Profile
+from datetime import date, timedelta
+
+# Get an instance of a logger
+logger = logging.getLogger(__name__)
+
+@shared_task
+def send_reminder_subscription_email():
+    profiles = Profile.objects.all()
```

Comment:
Use filter() to optimize SQL query
https://docs.djangoproject.com/fr/4.0/topics/db/queries/#field-lookups

Reply:
@raphaelmonnard Use django Q
@ioknox management command in on efile django